tools/checkpoint_conversion/convert_segformer_checkpoints.py

```diff
@@ -94,7 +94,14 @@ def main(_):
     )
     num_classes = 150 if "ade20k" in FLAGS.preset else 19
 
-    preprocessor = SegFormerImageSegmenterPreprocessor()
+    image_converter = keras_hub.layers.ResNetImageConverter(
```

Comment:
This needs to be `SegFormerImageConverter`
And we no longer use `mean` and `variance`,  you can pass in the below format
`SegFormerImageConverter(image_size=image_size, scale=scale, offset=offset)`